spec.md

```diff
@@ -1531,6 +1583,22 @@ To disallow infinitely recursive hardware, modules cannot contain instances of
 itself, either directly, or indirectly through instances of other modules it
 instantiates.
 
+## Invoke
+
+FIRRTL functions and intrinsics are invoked with the `invoke` statement.
+
+``` firrtl
+circuit Top :
+   function MyFunc : 
+     input a : UInt
+     output b : UInt
+   module MyModule :
+      input a: UInt
+      output b: UInt
+      b <= invoke MyFunc(a)
```

Comment:
what is the order of inputs to `MyFunc`? Should these be passed/connected-to by name?

Reply:
and if you have multiple outputs what does that look like on the LHS?